.drone.yml

```diff
@@ -0,0 +1,81 @@
+workspace:
+  base: /drone
+  path: src/github.com/owncloud/client
+
+branches:
+  - master
+  - 2.4
+  - 2.3
+
+clone:
+  git:
+    image: plugins/git
+    pull: true
+    tags: false
+
+pipeline:
+  prepare-clang:
+    image: owncloudci/client:latest
+    pull: true
+    environment:
+      - LC_ALL=C.UTF-8
+    commands:
+      - mkdir clang-build
+      - cd clang-build
+      - cmake -GNinja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_BUILD_TYPE="Debug" -DUNIT_TESTING=1 ..
+
+  building-clang:
+    image: owncloudci/client:latest
+    pull: true
+    environment:
+      - LC_ALL=C.UTF-8
+    commands:
+      - cd clang-build
+      - ninja -j4
+
+  testing-clang:
+    image: owncloudci/client:latest
+    pull: true
+    environment:
+      - LC_ALL=C.UTF-8
+    commands:
+      - cd clang-build
+      - ctest --output-on-failure
+
+  prepare-gcc:
+    image: owncloudci/client:latest
+    pull: true
+    environment:
+      - LC_ALL=C.UTF-8
+    commands:
+      - mkdir gcc-build
+      - cd gcc-build
+      - cmake -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_BUILD_TYPE="Debug" -DUNIT_TESTING=1 ..
```

Comment:
Is there a reason to use ninja with clang but not gcc?

Reply:
Yes, same as for the Jenkinsfile:

```
// We are building "Linux - GCC" with "make" and "Linux - Clang" with ninja,
// the combinations are more or less arbitrarily chosen. We just want to
// check that both compilers and both CMake generators work. It's
// unlikely that a specific generator only breaks with a specific
// compiler.
```

Reply:
Exactly.

Maybe the comment could be put back in there too so future reader will not wonder the same.

Reply:
I have added it to the top of the .drone.yml